.github/workflows/publish_to_pypi.yml

```diff
@@ -5,7 +5,32 @@ on:
     branches: [main]
     types:
       - closed
-name: Publishing to PyPI
-uses: pypa/gh-action-pypi-publish@release/v1
-  with:
-    password: "pypi-AgEIcHlwaS5vcmcCJGI0ODA3NThkLWYzODAtNDBhMi05MGJjLTZiNTNlMjIyZmJiNgACKlszLCJhZmRiYjYxNi1iMWFiLTQyNzgtODY3Yy0yODdmN2Y0Zjg0ODYiXQAABiA0dxgY72aE-3rXXseX7oAEIdKXCo83kcboKIlvqQ0VNA"
\ No newline at end of file
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution 📦 to PyPI
+      if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        password: "pypi-AgEIcHlwaS5vcmcCJGI0ODA3NThkLWYzODAtNDBhMi05MGJjLTZiNTNlMjIyZmJiNgACKlszLCJhZmRiYjYxNi1iMWFiLTQyNzgtODY3Yy0yODdmN2Y0Zjg0ODYiXQAABiA0dxgY72aE-3rXXseX7oAEIdKXCo83kcboKIlvqQ0VNA"
```

Comment:
We should password from secret. This repo will be open source.

Reply:
These are my personal account token, will use official secret key later.